lib/searchyll/mapping/fields.json

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
Is this file necessary? or maybe renamed as 'example'?

Reply:
@drusellers No this file isn't strictly necessary... maybe renamed as example is better.